webv2/web.go

```diff
@@ -1402,6 +1402,21 @@ func getDSpannerDDL(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(strings.Join(l, ""))
 }
 
+// getTypeDescription maps IssueDB's Category to corresponding CategoryDescription(if present),
+// or to the Brief if not present and pass the map to frontend to be used in assessment report UI
+func getTypeDescription(w http.ResponseWriter, r *http.Request) {
+	var CategoryMap = make(map[string]string)
```

Comment:
why is CategoryMap public , i.e. in caps?

Reply:
categorymap is very misleading , use IssuesMap or Issue Description

Reply:
Corrected it to issuesMap